.travis.yml

```diff
@@ -8,6 +12,22 @@ python:
 install:
   - pip install pep8
 before_script:
-   pep8 .
+  - pep8 .
 script:
   - python setup.py -q install
+jobs:
+  include:
+  - stage: deploy
+    python: '3.6'
+    install: skip
+    before_script: skip
+    script: skip
+    deploy:
+      stage: deploy
+      provider: pypi
+      on:
+        tags: true
+      distributions: sdist bdist_wheel
+      user: <PyPI user>
```

Comment:
There is  a such pypi user `reportportal`, so I would set `user: reportportal` (https://pypi.org/project/reportportal-client/) for now and @DzmitryHumianiuk  provided encrypted password for this user.